docs/user-stories/group-roles.md

```diff
@@ -0,0 +1,211 @@
+# Group Roles
+
+## Owner
```

Comment:
@stanislavkhatko Here are some group admin and owner stories.

Am I answering the right question with this document, or should I be trying something different?

Were you wanting more details on notifications here?